### DISCUSSION TITLE
[12.03] Service Suspension Notice 服务暂停公告

### DISCUSSION THREAD

**Discussion by wongself** [HammerPDF/Scholar-PDF-Reader#37] [Announcements]
Due to the malfunction of the air conditioning equipment in the server room, all academic functions of Hammer PDF are unavailable for protecting the device, and have been restored.
因服务器所在机房的空调设备故障，为了保护设备需立即停止运转，故 Hammer PDF 所有学术功能不可用，现已恢复。

**Comment by jd445** [comment]
现在是不是又不能用了呀

**Reply by wongself** [reply]
感谢您的提醒，我们正在调查学术服务运行情况。若有最新情况，我们会第一时间告知您。

**Reply by wongself** [reply]
目前学术服务已正常启动。
元旦期间服务器所在的大楼执行了两次计划停电，我们并没有在恢复供电后及时启动服务。
给您造成困扰，我们万分抱歉。

**Reply by jd445** [reply]
兄弟维护的挺及时啊.....辛苦辛苦
